src/Microsoft.DotNet.ProjectModel/Compilation/LibraryExporter.cs

```diff
@@ -166,8 +189,14 @@ private LibraryExport ExportProject(ProjectDescription project)
                 sourceReferences.Add(sharedFile);
             }
 
+            var contentFiles = project.Project.Files.GetContentFiles().Select(f => new LibraryContentFile()
```

Comment:
It needs to work for pack and publish. If you took this logic and made a nupkg the package and project would be different. That's what we need to avoid. We also need to change our "shared" to use this model.